int32 id
float32 risk_score
float32 collision_time
float32 distance
float32 velocity
